### ISSUE TITLE
How to disable click to add vertex to a polygon

### ISSUE THREAD

**Issue by vinhhungle** [freegroup/draw2d#209]
Currently, when clicking on a polygon line, it will add a vertex.
I'm trying to find a way to disable this behavior. Is there anyway to do this? Thank you for your help!

**Comment by freegroup** [comment]
everything in draw2d is managed by `EditPolicies` .

 - SelectionFeedback via rectangles, lines,...
 - DragDrop Behaviour
 - Copy&Paste
 - Interaction with Connections,
 - Zoom or Panning

and of course the edit behaviour of polygons. You can just install another EditPolicy to get the expected behaviour.

Example: https://codepen.io/freegroup/pen/PoEaRvj

**Comment by vinhhungle** [comment]
There is no policy related to click-to-add vertex behavior. However, I've found a work around. Thanks.

**Comment by vinhhungle** [comment]
Yes, I've seen it.  However, the requirement for my case is different. I still need the original vertices to be editable but I don't need the  click-to-add vertex behavior. So my current workaround is to add an attribute to the original vertices and remove any vertices without that attribute.

**Comment by freegroup** [comment]
the best is to implement your own SelectionHandler instead of removing the points after user-edit.
It is possible to manage almost everything via Policies.

I Update the example with your requirements

https://codepen.io/freegroup/pen/PoEaRvj

**Comment by vinhhungle** [comment]
Got it. Thank you for your help!
